src/routes/JobCreateRoute.js

```diff
@@ -75,13 +81,15 @@ class JobCreateRoute extends React.Component {
 
   render() {
     const { handlers, match: { params: { format } } } = this.props;
+    const localKB = this.props?.resources?.localKB?.records[0] || {};
```

Comment:
Keep consistent destructuring.

Also, `props` and `resources` are always defined so no need to optional chain them. `records` may not be though, so chain the indexing into the 0th entry.
```suggestion
    const localKB = props.resources.localKB?.records?.[0] ?? {};
```

Reply:
I'm not sure if there's a better way to destructure this than:

```const { handlers, match: { params: { format } }, resources } = this.props;
    const localKB = resources.localKB?.records?.[0] || {};```, so that's what I'm doing for now. Feel free to show me some better destructuring magic if it exists, a cursory google didn't seem to show anything